Gordon360/Models/ViewModels/AcademicCheckInViewModel.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace Gordon360.Models.ViewModels
+{
+    public class AcademicCheckInViewModel
+    {
+        public string Holds { get; set; }
+
+        public string personalPhone { get; set; }
+
+        public bool makePrivate { get; set; }
+
+        public bool noPhone { get; set; }
```

Comment:
Any reason why these are not capitalized?

Reply:
We wanted to make things consistent between the back and front end and those were originally written with camel case. We can change it if necessary but we'd have to mirror the change on the front end as well.

Reply:
Looks good now